.github/workflows/ci.yml

```diff
@@ -156,14 +156,31 @@ jobs:
         with:
           api-key: ${{ secrets.FORESIGHT_API_KEY }}
 
+  databases-new-store:
+    name: Databases New Store
+    runs-on: ubuntu-latest
+    steps:
+      - id: matrix-db
+        # language=bash
+        run: |
+          # For PRs, run only PostgreSQL, on branches and nightly runs run all databases
+          if [ "${GITHUB_EVENT_NAME}" == "pull_request" ]; then
```

Comment:
Nitpick, but I prefer to use ${{ github.event_name }} as that allows you to see the value when debugging, while using "${GITHUB_EVENT_NAME}" you don't see the value AFAIK

Reply:
Changes, and also rebased due to conflicts that occurred when other changes were merged into main.